packages/api/src/router/widgets/indexer-manager.ts

```diff
@@ -0,0 +1,67 @@
+import { TRPCError } from "@trpc/server";
+
+import { ProwlarrIntegration } from "@homarr/integrations";
+import { logger } from "@homarr/log";
+
+import { createManyIntegrationMiddleware } from "../../middlewares/integration";
+import { createTRPCRouter, publicProcedure } from "../../trpc";
+
+export const indexerManagerRouter = createTRPCRouter({
+  indexers: publicProcedure
+    .unstable_concat(createManyIntegrationMiddleware("query", "prowlarr"))
+    .query(async ({ ctx }) => {
+      const results = await Promise.all(
+        ctx.integrations.map(async (integration) => {
+          let client;
```

Comment:
Commented out code is bad practice. At this time, the second integration has not been implemented and it makes no sense to already partially implement related logic here.